GAE/src/locale/es.properties

```diff
@@ -145,154 +140,154 @@ Delete\ form = Borrar formulario
-Instance\ ID\ must\ be\ a\ number = El ID de la instancia debe ser numérico
-Invalid\ path = Ruta no válida
+Instance\ ID\ must\ be\ a\ number = El ID de la instancia debe ser num�rico
+Invalid\ path = Ruta no v�lida
 Item\ name\ (mandatory) = Nombre del elemento (obligatorio)
 Language = Idioma
-Last\ contact = Último contacto
-Last\ update = Última actualización
+Last\ contact = �ltimo contacto
+Last\ update = �ltima actualizaci�n
 Latitude = Latitud
-Leaving\ this\ page\ will\ cancel\ the\ upload\ currently\ in\ progress.\ Are\ you\ sure\ you\ want\ to\ do\ this? = Al salir de esta página cancelará la acción en progreso. ¿Está seguro de que quiere hacer esto?
+Leaving\ this\ page\ will\ cancel\ the\ upload\ currently\ in\ progress.\ Are\ you\ sure\ you\ want\ to\ do\ this? = Al salir de esta p�gina cancelar� la acci�n en progreso. �Est� seguro de que quiere hacer esto?
 Legend = Leyenda
 Length = Longitud
 Link = Enlace
 List\ of\ permissions = Lista de permisos
 Loading = Cargando
 Locate\ the\ data = Ubicar los datos
-Log\ in = Iniciar sesión
+Log\ in = Iniciar sesi�n
 Log\ out = Salir
 Longitude = Longitud
-Make\ data\ available\ for\ bulk\ upload\ using\ the\ Publish\ data\ option\ in\ the\ Akvo\ Flow\ app\ Settings. = Publique los datos para hacerlos disponibles desde la configuración de la aplicación Flow
+Make\ data\ available\ for\ bulk\ upload\ using\ the\ Publish\ data\ option\ in\ the\ Akvo\ Flow\ app\ Settings. = Publique los datos para hacerlos disponibles desde la configuraci�n de la aplicaci�n Flow
 Manage\ Cascade\ Resources = Administrar recursos cascada
 Manage\ Devices\ and\ Assignments = Administrar dispositivos y asignaciones
-Manage\ data\ approvals = Administrar la aprobación de datos
+Manage\ data\ approvals = Administrar la aprobaci�n de datos
 Manage\ device\ groups = Administrar grupos de dispositivos
 Manage\ notifications = Administrar notificaciones
 Manage\ translations = Administrar traducciones
 Mandatory = Obligatorio
 Manual\ survey\ transfer = Transferencia manual de encuesta
 Maps = Mapas
-Mark\ answers\ to\ this\ question\ as\ personal\ data\ (any\ data\ that\ can\ be\ used\ to\ identify\ a\ specific\ individual).\ When\ exporting\ the\ data\ via\ the\ Flow\ API,\ you\ can\ filter\ out\ the\ values\ based\ on\ this\ flag = Marcar respuestas a esta pregunta como datos personales (datos que se pueda usar para identificar a un individuo). Al exportar los datos a travs del Flow API, se puede filtrar los datos con en este indicador
+Mark\ answers\ to\ this\ question\ as\ personal\ data\ (any\ data\ that\ can\ be\ used\ to\ identify\ a\ specific\ individual).\ When\ exporting\ the\ data\ via\ the\ Flow\ API,\ you\ can\ filter\ out\ the\ values\ based\ on\ this\ flag = Marcar respuestas a esta pregunta como datos personales (datos que se pueda usar para identificar a un individuo). Al exportar los datos a trav\u008es del Flow API, se puede filtrar los datos con en este indicador
```

Comment:
trav\u008es: this is a bit strange, I see all the other chars represented correctly, ie í, ó shouldn't it be "través"?